k8s/src/test/scala/io/buoyant/k8s/IngressCacheTest.scala

```diff
@@ -1,12 +1,228 @@
 package io.buoyant.k8s
 
+import com.twitter.finagle.Service
+import com.twitter.io.Buf
+import com.twitter.util.Future
 import io.buoyant.test.Awaits
 import org.scalatest.FunSuite
+import com.twitter.finagle.http.{Request => FRequest, Response => FResponse}
```

Comment:
In general, I think our convention is to rename imports only when necessary. If you were importing both `finagle.http.Request` and a type from Linkerd called `Request`, then I think we would rename the type from Finagle. But in most of our code that I've seen where we don't have to deal with namespace clashes, we tend not to rename imports by default.